app/services/foreman_resource_quota/resource_origins/compute_attributes_origin.rb

```diff
@@ -0,0 +1,64 @@
+# frozen_string_literal: true
+
+module ForemanResourceQuota
+  module ResourceOrigins
+    class ComputeAttributesOrigin < ResourceOrigin
+      def host_attribute_eager_name
+        :compute_attributes
+      end
+
+      def host_attribute_name
+        :compute_attributes
+      end
+
+      def extract_cpu_cores(param)
+        return nil unless param.key?(:cpus)
+        param[:cpus].to_i
+      rescue StandardError
+        nil
+      end
+
+      def extract_memory_mb(param)
+        case determine_memory_key(param)
+        when :memory
+          param[:memory].to_i / ResourceQuotaHelper::FACTOR_B_TO_MB
+        when :memory_mb
+          param[:memory_mb].to_i
+        end
+      rescue StandardError
+        nil
+      end
+
+      def extract_disk_gb(param)
+        return nil unless param.key?(:volumes_attributes)
+        extract_volumes(param).sum do |disk|
+          # HERE: Has size_gb here too!!
```

Comment:
What does this mean?

Reply:
A reminder for myself which should be removed, thanks! (TODO)